.readthedocs.yaml

```diff
@@ -0,0 +1,19 @@
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "mambaforge-4.10"
+
+python:
```

Comment:
Is this stanza necessary? I don't see it in the [template](https://github.com/neutrons/python_project_template/blob/main/.readthedocs.yaml) which makes me curious